nrcemt/qeels/engine/peak_detection.py

```diff
@@ -67,3 +68,89 @@ def rotate_points(x1, y1, x2, y2, rotation_angle_rad, width, height):
     y2_rotated = y2_width + y2_height + width/2
 
     return (x1_rotated, y1_rotated, x2_rotated, y2_rotated)
+
+
+# Section commented out because if "peak" is on very edge
+# of ycfit it will not be the returned value
```

Comment:
add more context, explaining the old usage of find peaks essentially just found the max